Client/Ecosia/L10N/String.swift

```diff
@@ -56,6 +56,9 @@ extension String {
         case networkError = "Network error!"
         case new = "New"
         case noConnectionMessage = "We couldn't verify your link. Please check your internet connection and try again."
+        case noConnectionNSURLErrorTitle = "No connection"
```

Comment:
I see that you introduced those three string. I wonder if they are in some form existing in FF's l10n already. If so we don't need to provide our own translation and use the given ones, which in this case are missing. But I am happy to explain to you how to get those 🚀